manifests/m/Microsoft/Git/2.32.0.0.0/Microsoft.Git.yaml

```diff
@@ -0,0 +1,22 @@
+PackageIdentifier: Microsoft.Git
+PackageVersion: 2.32.0.0.0
+PackageName: Microsoft Git
+Publisher: The Git Development Community
+Moniker: microsoft-git
+PackageUrl: https://aka.ms/ms-git
+Tags: [ microsoft-git ]
+License: Copyright (C) Microsoft Corporation
+ShortDescription: |
+  Git distribution to support monorepo scenarios.
+  Note: This is not Git for Windows. Unless you are working in a monorepo and require
+  specific Git modifications, please run `winget install git` to start using Git for Windows.
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/microsoft/git/releases/download/v2.32.0.vfs.0.0/Git-2.32.0.vfs.0.0-64-bit.exe
+  InstallerType: inno
+  InstallerSha256: 73fe3e86ae419b7f63f87635850dc5d3c8285379e28b765e67501e502d4588b1
+  InstallerSwitches:
+    Custom: /COMPONENTS="AUTOUPDATE"
```

Comment:
@ldennington We should remove these `InstallerSwitches` now that we've adjusted the installer defaults.

Reply:
Will do - I'm going to do a larger manifest update once https://github.com/mjcheetham/update-winget/pull/124 merges and a new release of the update-winget task is cut. I will be sure to include at that time, and will manually make the update for now.

Reply:
Oh never mind - thought this was the new PR against one of the new versions. Will not manually update :).